### PR TITLE
qt: Prefix makefile variables with QT_

### PR DESCRIPTION
**Pull request by barton2526**
> Improves consistency and readability if future QML variables are added.

Ref: https://github.com/bitcoin/bitcoin/pull/23242

### CODE REVIEW

**Review by iFoggz — APPROVED** [review]
Concept ACK 

Side note: I don't believe we use QT_RES_MOVIES

File: src/Makefile.qt.include [hunk 1]
```diff
@@ -597,18 +597,18 @@ RES_IMAGES = \
...
   qt/res/fonts/Inter-Regular.otf
 
-RES_STYLESHEETS = \
+QT_RES_STYLESHEETS = \
   qt/res/stylesheets/light_stylesheet.qss \
   qt/res/stylesheets/dark_stylesheet.qss
 
 
-GRIDCOIN_RC = qt/res/gridcoinresearch.rc
+GRIDCOIN__QT_RC = qt/res/gridcoinresearch.rc
```
Comment (Pythonix) [inline src/Makefile.qt.include]: > GRIDCOIN__QT_RC

Here seems to have been inserted a underscore too much?

Comment (jamescowens) [reply src/Makefile.qt.include]: I agree. Not sure how it is working with that...

Comment (barton2526) [reply src/Makefile.qt.include]: > > GRIDCOIN__QT_RC
> 
> Here seems to have been inserted a underscore too much?

Good catch, thanks!

**Review by jamescowens — APPROVED** [review]
utACK

**Pull request merged**